src/main/java/org/baggle/domain/meeting/domain/Meeting.java

```diff
@@ -67,4 +60,13 @@ public void updateMeetingStatusIntoOngoing() {
     public void updateMeetingStatusIntoTermination() {
         this.meetingStatus = MeetingStatus.TERMINATION;
     }
+
+    public void updateParticipationList(Participation newParticipation) {
+        this.participations.forEach(participation -> participation.updateButtonAuthority(ButtonAuthority.NON_OWNER));
+        this.participations.add(newParticipation);
+    }
```

Comment:
하나의 함수에서 두 가지의 역할을 하고 있다는 느낌을 받았습니다. 아래와 같이 두 가지 함수로 분리하면 어떨까요?
```suggestion
    public void updateButtonAuthorityOfParticipations(ButtonAuthority buttonAuthority) {
        getParticipations().forEach(participation -> participation.updateButtonAuthority(buttonAuthority));
    }
    
    public void addParticipation(Participation newParticipation) {
        getParticipations().add(newParticipation);
    }
```

Reply:
맞는 말씀인 것 같습니다!! 혹시 this.participations 말고 getParticiaptions() 메서드를 사용한 이유를 알 수 있을까요...?